opm/parser/eclipse/EclipseState/Eclipse3DProperties.cpp

```diff
@@ -449,6 +449,17 @@ namespace Opm {
         }
     }
 
+    std::vector< int > Eclipse3DProperties::getRegions( const std::string& kw ) const {
```

Comment:
They're not *values* though, they're names/identifiers represented as an integer.